k8s/lib.sh

```diff
@@ -94,6 +94,16 @@ k8s::remove::containers() {
   done
 }
 
+k8s::remove::containerd() {
+  k8s::common::setup_env
+
+  # only remove containerd if the snap was already bootstrapped.
+  # this is to prevent removing containerd when it is not installed by the snap.
+  if [ -f "$SNAP_COMMON/lock/containerd-socket-path" ]; then
+     rm -f $(cat "$SNAP_COMMON/lock/containerd-socket-path")
+  fi
+}
```

Comment:
If this is removing a directory, wouldn't you need `rm -rf`? Also, if this is the containerd directory path, maybe `containerd-socket-path` is not the most accurate name?